src/OrchardCore/OrchardCore.Email.Core/Services/SmtpEmailService.cs

```diff
@@ -0,0 +1,224 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Net.Security;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+using System.Threading.Tasks;
+using MailKit.Net.Proxy;
+using MailKit.Net.Smtp;
+using MailKit.Security;
+using Microsoft.Extensions.Localization;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using MimeKit;
+
+namespace OrchardCore.Email.Services
+{
+    /// <summary>
+    /// Represents a SMTP service that allows to send emails.
+    /// </summary>
+    /// <param name="options">The <see cref="IOptions{SmtpSettings}"/>.</param>
+    /// <param name="logger">The <see cref="ILogger{SmtpService}"/>.</param>
+    /// <param name="stringLocalizer">The <see cref="IStringLocalizer{SmtpService}"/>.</param>
+    /// <param name="emailAddressValidator">The <see cref="IEmailAddressValidator"/>.</param>
+    public class SmtpEmailService(
+        IOptions<SmtpEmailSettings> options,
+        ILogger<SmtpEmailService> logger,
+        IStringLocalizer<SmtpEmailService> stringLocalizer,
+        IEmailAddressValidator emailAddressValidator) : EmailServiceBase<SmtpEmailSettings>(options, logger, stringLocalizer, emailAddressValidator)
+    {
+        private const string EmailExtension = ".eml";
+
+        /// <inheritdoc/>
+        /// <remarks>This method allows to send an email without setting <see cref="MailMessage.To"/> if <see cref="MailMessage.Cc"/> or <see cref="MailMessage.Bcc"/> is provided.</remarks>
+        public override async Task<EmailResult> SendAsync(MailMessage message)
+        {
+            ArgumentNullException.ThrowIfNull(message);
+
+            if (Settings == null)
+            {
+                return EmailResult.Failed(S["SMTP settings must be configured before an email can be sent."]);
+            }
+
+            SmtpEmailResult result;
+            var response = default(string);
+            try
+            {
+                // Set the MailMessage.From, to avoid the confusion between Options.DefaultSender (Author) and submitter (Sender)
```

Comment:
```suggestion
                // Set the MailMessage.From, to avoid the confusion between Options.DefaultSender (Author) and submitter (Sender).
```